configs/mobilevit/mobilevit-small_8xb128_in1k.py

```diff
@@ -0,0 +1,31 @@
+_base_ = [
+    '../_base_/models/mobilevit/mobilevit_s.py',
+    '../_base_/datasets/imagenet_bs32.py',
+    '../_base_/default_runtime.py',
+    '../_base_/schedules/imagenet_bs256.py',
+]
+
+# no normalize for original implements
+# use bgr directly
```

Comment:
move this line above the `to_rgb=False`.

Reply:
done